install/installer/pkg/config/v1/config.go

```diff
@@ -218,13 +218,12 @@ type DatabaseCloudSQL struct {
 }
 
 type ObjectStorage struct {
-	InCluster          *bool                      `json:"inCluster,omitempty"`
-	S3                 *ObjectStorageS3           `json:"s3,omitempty"`
-	CloudStorage       *ObjectStorageCloudStorage `json:"cloudStorage,omitempty"`
-	Azure              *ObjectStorageAzure        `json:"azure,omitempty"`
-	MaximumBackupCount *int                       `json:"maximumBackupCount,omitempty"`
```

Comment:
This is a backwards incompatible change, afaik we'd break anyone who has a `maximumBackupCount` entry in their installer config.

We could leave the field and mark it as deprecated (a comment and validation). WDYT?

Reply:
Done